src/schema/v2/Feature/FeatureType.ts

```diff
@@ -34,5 +37,16 @@ export const FeatureType = new GraphQLObjectType<
         return feature
       },
     },
+    sets: {
```

Comment:
I dunno if we wanna name this `sections` or just alias it?

Reply:
I don't mind keeping this naming.